frontend/src/static/js/v_summary.js

```diff
@@ -117,27 +117,40 @@ window.vSummary = {
       minDate: '',
       maxDate: '',
       contributionBarColors: {},
+      isWatcherUpdated: false, // keep track of whether first cycle of watcher updates is finished
     };
   },
   watch: {
     repos() {
-      this.getFiltered();
+      if (this.isWatcherUpdated) {
+        this.getFiltered();
+      }
```

Comment:
Should this be method with a descriptive name?

Reply:
Yes I think it would be better to make this as a method.